core/src/main/java/org/mskcc/cbio/portal/model/Sample.java

```diff
@@ -52,15 +52,15 @@
         PRIMARY_BLOOD_TUMOR("Primary Blood Tumor"),
         RECURRENT_BLOOD_TUMOR("Recurrent Blood Tumor"),
         METASTATIC("Metastatic"),
-        BLOOD_NORMAL("Blood Derived Normal"),
-        SOLID_NORMAL("Solid Tissues Normal");
+        BLOOD_DERIVED_NORMAL("Blood Derived Normal"),
+        SOLID_TISSUES_NORMAL("Solid Tissues Normal");
 
         private String propertyName;
         
         Type(String propertyName) { this.propertyName = propertyName; }
         public String toString() { return propertyName; }
-        public boolean isNormal() { return this==BLOOD_NORMAL || this==SOLID_NORMAL; }
-        public static Set<Type> normalTypes() {return EnumSet.of(BLOOD_NORMAL, SOLID_NORMAL);}
+        public boolean isNormal() { return this==BLOOD_DERIVED_NORMAL || this==SOLID_TISSUES_NORMAL; }
+        public static Set<Type> normalTypes() {return EnumSet.of(BLOOD_DERIVED_NORMAL, SOLID_TISSUES_NORMAL);}
```

Comment:
Spacing inconsistencies, the line above it has a space. Just being picky. It was like that before but might as well fix it here if you want to.